docs/tutorials/tuners/parameter_grid.rst

```diff
@@ -0,0 +1,162 @@
+Tuning a DFFML model with ParameterGrid
```

Comment:
Let's add both these tutorials to the line in the CI YAML file here: https://github.com/intel/dffml/blob/748c1b9072a1a2f4f8583e6f432792ad5cb17857/.github/workflows/testing.yml#L219